app.rb

```diff
@@ -31,19 +32,74 @@
 config_overrides[:port] = ENV['APRUVE_PORT'] unless ENV['APRUVE_PORT'].nil?
 Apruve.configure(ENV['APRUVE_API_KEY'], apruve_environment, config_overrides)
 merchant_id = ENV['APRUVE_MERCHANT_ID']
+corporate_account_id = ENV['APRUVE_CORPORATE_ACCOUNT_ID']
 
 before '/webhook_notify' do
   request.body.rewind
   @webhook_body = request.body.read
 end
 
+get '/offline_order' do
+  erb :offline_orders
+end
+
+get '/customers' do
+  content_type :json
+  send_file File.expand_path('customers.json', settings.public_folder)
```

Comment:
The problem with the JSON file is that the IDs only apply to one environment.  You can populate it with IDs for staging or you can populate it with IDs for test, but it's only ever going to work on one environment.

I'd say the updated library is a requirement for this.